_data/sources/meetup.js

```diff
@@ -1,17 +1,63 @@
-const fetch = require("node-fetch"),
+const {GraphQLClient, gql} = require("graphql-request"),
     group = require("./templates/group"),
     event = require("./templates/event");
 
 class meetup {
-    constructor() {
+    constructor(token = process.env.MEETUP_TOKEN) {
         this.groups = require("./groupIds/meetup").filter((value, index, self) => self.indexOf(value) === index);
-        this.apiEvents = (group) => 'https://api.meetup.com/' + group + '/events';
+        this.graphQLClient = new GraphQLClient('https://api.meetup.com/gql', {
+            headers: {
+                authorization: 'Bearer ' + token,
+                method: 'POST',
+                dataType: 'jsonp'
+            },
+        });
+        this.apiEvents = (group) => gql`
+            {
+                groupByUrlname(urlname: ${group}) {
+                    unifiedEvents {
+                        edges {
+                            node {
+                                title
+                                shortUrl
+                                venue
+                                description
+                                dateTime
+                                endTime
+                                going
+                                maxTickets
+                                fees
+                                price
+                                groupName
+                                groupLink
+                                source
+                                ad
+                            }
+                        }
+                    }
+                    name
+                    link 
+                }
+            }
+        `;
         this.apiPastEvents = (group) => 'https://api.meetup.com/' + group + '/events?desc=true&status=past';
-        this.apiGroup = (group) => 'https://api.meetup.com/' + group;
-        this.header = {
-            method: 'GET',
-            dataType: 'jsonp'
-        };
+        this.apiGroup = (group) => gql`
+            {
+                groupByUrlname(urlname: ${group}) {
+                    id
+                    name
+                    desc
+                    link
+                    links
+                    img
+                    members
+                    sinceLast
```

Comment:
I do not believe this property exists naively to either API, for clarification (until I document it) this is meant to be a positive Integer representing the amount of days since the groups last event, if there are no past events (new groups, for example), this property will be `null`, `undefined` or `false` (any falsely evaluated value).